index.html

```diff
@@ -578,148 +487,15 @@ <h3>The <dfn>options</dfn> attribute</h3>
   <h3>The <dfn>ComputePressureObserverOptions</dfn> dictionary</h3>
```

Comment:
I believe you can now remove it

Reply:
we want to reuse later and it is even good for future proofing

Reply:
Maybe add a comment inside explaining the current status.
New people may look at the spec and won't understand this.

Reply:
Will do!